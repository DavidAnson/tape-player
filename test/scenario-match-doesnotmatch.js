// @ts-check

"use strict";

const tape = require("tape");
require("../index.js");

tape("match test", (test) => {
  test.match("string", /s.r.n./u);
  test.match("string", /s.r.n./u, "srn message");
  test.match("string", /n.t/u);
  test.match("string", /n.t/u, "nt message");
  test.end();
});

tape("doesNotMatch test", (test) => {
  test.doesNotMatch("string", /n.t/u);
  test.doesNotMatch("string", /n.t/u, "nt message");
  test.doesNotMatch("string", /s.r.n./u);
  test.doesNotMatch("string", /s.r.n./u, "srn message");
  test.end();
});
