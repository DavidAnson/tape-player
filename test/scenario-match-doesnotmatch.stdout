match test
  scenario-match-doesnotmatch.js:11:8
  Message:  The input did not match the regular expression /n.t/u. Input: 'string'
  Operator: match
  Expected: /n.t/u
  Actual:   'string'
  scenario-match-doesnotmatch.js:12:8
  Message:  nt message
  Operator: match
  Expected: /n.t/u
  Actual:   'string'
doesNotMatch test
  scenario-match-doesnotmatch.js:19:8
  Message:  The input was expected to not match the regular expression /s.r.n./u. Input: 'string'
  Operator: doesNotMatch
  Expected: /s.r.n./u
  Actual:   'string'
  scenario-match-doesnotmatch.js:20:8
  Message:  srn message
  Operator: doesNotMatch
  Expected: /s.r.n./u
  Actual:   'string'

Tests:    2
Asserts:  8
Failures: 4
